buildkite/src/Constants/Network.dhall

```diff
@@ -23,19 +29,31 @@ let lowerName =
       ->  merge
             { Devnet = "devnet"
             , Mainnet = "mainnet"
-            , Berkeley = "berkeley"
+            , TestnetGeneric = "testnet_generic"
             , DevnetLegacy = "devnet_legacy"
             , MainnetLegacy = "mainnet_legacy"
             , PreMesa1 = "hetzner-pre-mesa-1"
             }
             network
 
+let debianSuffix =
```

Comment:
So now we have yet another function to represent labels as kebab case.. However lowerName is still used across codebase to build debian packages it seems

Reply:
If possible I'd suggest remove lowerName and replace their occurence with debianSuffix.